File: support_tools/swigwin-3.0.12/Examples/test-suite/php5/cpp_static_runme.php
<?php

require "tests.php";
require "cpp_static.php";

// New functions
check::functions(array(staticfunctiontest_static_func,staticfunctiontest_static_func_2,staticfunctiontest_static_func_3,is_python_builtin));
// New classes
check::classes(array(StaticMemberTest,StaticFunctionTest,cpp_static,StaticBase,StaticDerived));
// New vars
check::globals(array(staticmembertest_static_int,staticbase_statty,staticderived_statty));

check::done();
?>

File: support_tools/swigwin-3.0.12/Examples/test-suite/go/cpp_static_runme.go
package main

import . "./cpp_static"

func main() {
	StaticFunctionTestStatic_func()
	StaticFunctionTestStatic_func_2(1)
	StaticFunctionTestStatic_func_3(1, 2)
	SetStaticMemberTestStatic_int(10)
	if GetStaticMemberTestStatic_int() != 10 {
		panic(GetStaticMemberTestStatic_int())
	}
}
